test/scripts/e2e-cp.sh

```diff
@@ -6,6 +6,24 @@ set -ex
 export KUBERNETES_CONFORMANCE_TEST=y
 export KUBECONFIG=${HOME}/admin.conf
 
+IPV6_SKIPPED_TESTS="Should be allowed by externalip services|\
+should provide connection to external host by DNS name from a pod|\
+should provide Internet connection continuously when master is killed|\
+should provide Internet connection continuously when ovn-k8s pod is killed|\
+Should validate connectivity from a pod to a non-node host address on same node|\
+Should validate connectivity to an external gateway\'s loopback address via a pod with external gateway annotations enabled|\
+Should validate connectivity to multiple external gateways for an ECMP scenario|\
+Should validate connectivity without vxlan before and after updating the namespace annotation to a new external gateway|\
+Should validate ICMP connectivity to an external gateway\'s loopback address via a pod with external gateway annotations enabled|\
+Should validate ICMP connectivity to multiple external gateways for an ECMP scenario|\
+Should validate ingress connectivity from an external gateway|\
+Should validate NetFlow data of br-int is sent to an external gateway|\
+Should validate TCP/UDP connectivity to an external gateway\'s loopback address via a pod with external gateway annotations enabled|\
+Should validate TCP/UDP connectivity to multiple external gateways for a UDP / TCP scenario|\
+Should validate the egress firewall policy functionality against remote hosts|\
+Should validate the egress IP functionality against remote hosts|\
+recovering from deleting db files while maintain connectivity"
```

Comment:
I could add a link to this PR here, so the reader can check, why these tests are skipped currently.

Reply:
that makes sense. We should also find out which of these are important enough that we want to push for getting them fixed. I'll bring it up elsewhere though. This PR seems good to me as is, but adding a link to this PR is probably worth it.

Reply:
Thanks @jluhrsen. I added a link to this PR.